src/components/Nav/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Precisamos mesmo do subdiretório Nav? Caso contrário, vamos prosseguir colocando a maior parte da coisas na raiz e só criando diretórios quando necessário.  Não há problema em cirar subdiretórios, porém acho que devemos ir criando quando for necessário.

Reply:
Eu acredito que sim, pois ele é uma partial (componente reutilizável) e vou estar usando ele em mais de uma page, e como eu uso esse padrão modular de pastas pra conseguir deixar os partials mais reutilizáveis eu acho que é um padrão interessante, aí tipo, importante citar tbm que o nav é reutilizado em N lugares na aplicação, então não torna-lo um partial seria uma reutilização desnecessária de código

Reply:
Duas perguntas diretas. Caso prefira, podemos conversar via fone ou VC.

- Os diretórios são **necessários** para criação de componnte?
- Quais as vantagens de usar subdiretórios em Next.js?

Outro ponto interessante:
-  Quando comparo com [esse](https://github.com/mmmeff/nextjs-page-template-example) projeto template para Next.js, noto que não precisa do diretório src.

Reply:
Se o senhor quiser podemos marcar uma pequena reunião para eu tirar essas dúvidas mais estruturais, se o senhor tiver tempo é claro